modules/andes-core/broker/src/main/java/org/wso2/andes/store/cassandra/HectorBasedAndesContextStoreImpl.java

```diff
@@ -672,6 +672,13 @@ public void close() {
     }
 
     /**
+     * {@inheritDoc}
+     */
+    @Override public void clearSlotStorage() throws AndesException {
+        throw new NotImplementedException();
+    }
+
```

Comment:
Phrase "Not implemented" gives you a feeling that this is something that needs to be implemented but hasn't been done so yet. What if we use "Not Supported" instead? WDYT?